api/turing/api/appcontext.go

```diff
@@ -190,20 +189,26 @@ func NewAppContext(
 // getEnvironmentClusterMap creates a map of the environment name to the kubernetes cluster. Additionally,
```

Comment:
This comment still refers to the old method

Reply:
Fixed